src/vision/status_quo/frederik_makes_their_first_steps_into_async.md

```diff
@@ -0,0 +1,64 @@
+
+### Learning about Async
+Still enarmored with the first edition of the Rust book, they decide to go looking for an updated version, hoping that it will teach them async in the same manner that it taught them so much about the language and design patterns for Rust. Dissapointed, they find no mention of async in the book, aside from a note that it exists as a keyword.
+
+Not to be deterred, they go looking further, and start looking for similarly great documentation about async.
+After stumbling upon the async book, their dissapointment is briefly replaced with relief as the async book does a good job at solidifying what they have already learned in various blog posts about async, why one would use it and even a bit about how it all works under the hood.
+They skim over the parts that seem a bit too in depth for now like pinning, as they're looking to quickly get their hands dirty.
+Chapter 8: The Async Ecosystem teaches them what they already picked up on through blog posts and contentious tweets: the choice of the runtime has large implications on what libraries they can use.
+
+### The wrong time for big decisions
+Frederik's dreams to quickly get their hands dirty with async Rust are shattered as they discover that they first need to make a big choice: what executor to use. Having had quite a bit of exposure to the conversations surrounding the incompatible ecosystems, Frederik is perhaps a bit more paranoid about making the wrong choice than the average newcomer.
+This feels like a big decision to them, as it would influence the libraries they could use and switching to a different ecosystem would be all but impossible after a while. Since they would like to choose what libraries they use before having to choose an executor, Frederik feels like the decision making is turned on its head. 
+
+Their paranoia about choosing the right ecosystem is eased after a few days of research, and some more conversations on the Rust subreddit, after which they discover that most of the RPC libraries they might want to use are situated within the most popular Tokio ecosystem anyways. Tokio also has a brief tutorial, which teaches them some basic concepts within Tokio and talks a bit more about async in general.
+
+### Woes of a newcomer to async
+Being reasonably confident in their choice of ecosystem, Frederik starts building their distributed system.
+After a while, they want to introduce another networking library of which the api isn't async. Luckily Frederik picked up on that blocking was not allowed in async (or at least not in any of the currently existing executors), through reading some blog posts about async. More reddit discussions point them towards spawn\_blocking in Tokio, and even rayon. But they're none the wiser about how to apply these paradigms in a neat manner.
```

Comment:
It's worth noting that this knowledge came through blog posts and not through documentation or any sort of hint from the compiler.